src/main/java/com/epam/spring/homework3/controller/ClientController.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.spring.homework3.controller;
+
+import com.epam.spring.homework3.dto.ClientDTO;
+import com.epam.spring.homework3.service.ClientService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/api")
```

Comment:
All controllers have base path of _''./api'"_. That stuff can be moved out of controllers, and stored in one single place - _application.properties_ file. [Here are the examples](https://www.baeldung.com/spring-boot-context-path)